### DISCUSSION TITLE
set inav_allow_gps_fix_estimation=ON is rejected by CLI INAV 7.1.2

### DISCUSSION THREAD

**Discussion by omesen** [iNavFlight/inav#10319] [General]
This command set inav_allow_gps_fix_estimation=ON is rejected by CLI, I'm looking for a substitute to see if it exists. Does anyone know another option for this?
https://github.com/iNavFlight/inav/blob/master/docs/GPS_fix_estimation.md

**Comment by mmosca** [comment]
This is a new feature. If you want to test it, use a nightly build of inav 8.0.0
